styles/style.css

```diff
@@ -0,0 +1,571 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+  box-sizing: border-box;
+  color: #272a31;
```

Comment:
- [ ]  You should use both Lato and Cocogoose as your font-family. seemed you haven't used Cocogoose font family in your design at all. please used the provided fonts to resemble your website to the one from behance design.